### ISSUE TITLE
init wants to install all of anaconda

### ISSUE THREAD

**Issue by AlbertDeFusco** [Anaconda-Platform/anaconda-project#148]
When I run `anaconda-project init` my spec always includes the `anaconda` distribution package. Can this be disabled without having to run `anaconda-project remove-packages` or editing the yml file first?

All I wanted to do was create a minimal `anaconda-project.yml` add `python=3` and `pandas` to create a lightweight CLI project.

```
>mkdir test-proj
>cd test-proj
>anaconda-project init
Project configuration is in /Users/adefusco/AnacondaProjects/test/anaconda-project.yml
>time anaconda-project prepare
Solving environment: ...working... done

## Package Plan ##

  environment location: /Users/adefusco/AnacondaProjects/test/envs/default

  added / updated specs: 
    - anaconda

...OUTPUT TRIMMED...

The project is ready to run commands.
Use `anaconda-project list-commands` to see what's available.

real	6m10.071s
user	4m43.097s
sys	0m37.890s
```

## versions

```
>anaconda-project --version
0.8.2
```

```
>conda info

     active environment : base
    active env location : /Users/adefusco/Applications/miniconda3
            shell level : 1
       user config file : /Users/adefusco/.condarc
 populated config files : /Users/adefusco/Applications/miniconda3/.condarc
                          /Users/adefusco/.condarc
          conda version : 4.4.8
    conda-build version : 3.2.1
         python version : 3.6.4.final.0
       base environment : /Users/adefusco/Applications/miniconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/main/osx-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
          package cache : /Users/adefusco/Applications/miniconda3/pkgs
                          /Users/adefusco/.conda/pkgs
       envs directories : /Users/adefusco/Applications/miniconda3/envs
                          /Users/adefusco/.conda/envs
               platform : osx-64
             user-agent : conda/4.4.8 requests/2.18.4 CPython/3.6.4 Darwin/17.3.0 OSX/10.13.2
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

**Comment by AlbertDeFusco** [comment]
So, I found out something about why things seemed weird to me. I had attempted to add a file download *before* specifying packages. 

## packages first
Adding `python=3 pandas` means that only those dependencies need to be satisfied. 

```
>anaconda-project init
>anaconda-project add-packages python=3 pandas
...
The following NEW packages will be INSTALLED:

    anaconda:        custom-py36ha4fed55_0
    ca-certificates: 2017.08.26-ha1e5d58_0
    certifi:         2018.1.18-py36_0     
    intel-openmp:    2018.0.0-h8158457_8  
    libcxx:          4.0.1-h579ed51_0     
    libcxxabi:       4.0.1-hebd6815_0     
    libedit:         3.1-hb4e282d_0       
    libffi:          3.2.1-h475c297_4     
    libgfortran:     3.0.1-h93005f0_2     
    mkl:             2018.0.1-hfbd8650_4  
    ncurses:         6.0-hd04f020_2       
    numpy:           1.14.0-py36h8a80b8c_1
    openssl:         1.0.2n-hdbc3d79_0    
    pandas:          0.22.0-py36h0a44026_0
    pip:             9.0.1-py36h1555ced_4 
    python:          3.6.4-hc167b69_1     
    python-dateutil: 2.6.1-py36h86d2abb_1 
    pytz:            2017.3-py36hf0bf824_0
    readline:        7.0-hc1231fa_4       
    setuptools:      38.4.0-py36_0        
    six:             1.11.0-py36h0e22d5e_1
    sqlite:          3.22.0-h3efe00b_0    
    tk:              8.6.7-h35a86e2_3     
    wheel:           0.30.0-py36h5eb2c71_1
    xz:              5.2.3-h0278029_2     
    zlib:            1.2.11-hf3cbc9b_2
...
```


## download first
If I attempt to do anything but `add-packages` it seems like it defaults to creating a complete Anaconda install.

```
>anaconda-project init
>anaconda-project add-download IRIS https://raw.githubusercontent.com/bokeh/bokeh/f9aa6a8caae8c7c12efd32be95ec7b0216f62203/bokeh/sampledata/iris.csv
Solving environment: ...working...
...
  added / updated specs: 
    - anaconda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    numexpr-2.6.2              |   py36h0f4f1da_1         120 KB
    llvmlite-0.20.0            |           py36_0         9.2 MB
    filelock-2.0.12            |   py36h0d0b4fb_0          11 KB
    pycurl-7.43.0              |   py36hdb90038_3          57 KB
    pandas-0.20.3              |   py36hd6655d8_2         9.2 MB
    mpc-1.0.3                  |       hc455b36_4          88 KB
...
```

**Comment by mcg1969** [comment]
@AlbertDeFusco what you're seeing above is likely some sort of weird interaction with the `anaconda-custom` package. This is a special metapackage we created that resolved some undesirable user experiences when trying to customize individual packages in an environment containing Anaconda. Unfortunately, it sometimes results in anaconda not being installed at all.

What I've done for now in #178 is make it possible to avoid adding `anaconda` to the default environment in the first place.

**Comment by AlbertDeFusco** [comment]
fixed by PR #322
